sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_do_while.py

```diff
@@ -2,9 +2,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 from azure.ai.ml.entities._builders.do_while import DoWhile
+from azure.ai.ml.exceptions import UserErrorException
 
 
-def do_while(body, condition, mapping, *, max_iteration_count=None):
+def do_while(body, mapping, max_iteration_count: int, condition=True):
```

Comment:
default None or True?

Reply:
I think is `True`, otherwise when user specifies `do_while(max_iteration_count=<max_iteration_count>)`, we still need extra code assign value `True` for `condition`.

Reply:
I would prefer None. @zhengfeiwang  @D-W-  we can have a offline chat on this

Reply:
Offline synced on this, use `None` for: 1) aligned experience between SDK/CLI; 2) avoid confusing port name in body.